core/lib/spree/core/permalinks.rb

```diff
@@ -44,21 +44,18 @@ def permalink_order
       end
 
       def generate_permalink
-        "#{self.class.permalink_prefix}#{Array.new(self.class.permalink_length){rand(9)}.join}"
+        permalink_value = "#{self.class.permalink_prefix}#{Array.new(self.class.permalink_length){rand(9)}.join}"
```

Comment:
I don't think you'll need this variable setting, since this method just returns the string